solutions/devsprint-pedro-menezes-1/DeliveryAppChallenge/Modules/Product/Scenes/Carol/CAIsEmailValidUseCase.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  IsEmailValidUseCase.swift
+//  DeliveryAppChallenge
+//
+//  Created by Carolina Quiterio on 27/07/22.
+//
+
+import Foundation
+
+class CAIsEmailValidUseCase {
+    
+    func simpleValidation(_ email: String?) -> Bool {
```

Comment:
sugeriria `isEmailValid(for email: String)`

Reply:
Utilizei esse `simpleValidation` porque quero colocar uma validação de email mais complexa que é utilizada em outros lugares nesse UseCase também. Aí se fosse `isEmailValid` ficaria muito genérico. Oq acha?

Reply:
perfeito